bundles/af-core/pom.xml

```diff
@@ -159,7 +159,7 @@
                                                 <limit>
                                                     <counter>INSTRUCTION</counter>
                                                     <value>COVEREDRATIO</value>
-                                                    <minimum>0.80</minimum>
+                                                    <minimum>0.60</minimum>
```

Comment:
pls log a jira for this.

Reply:
https://jira.corp.adobe.com/browse/CQ-4351769